apps/learninglocker/templates/services/app/configs/ecosystem.config.js.j2

```diff
@@ -0,0 +1,19 @@
+module.exports = [{
```

Comment:
curious about the name of the file containing `ecosystem` and not 'pm2`.

Reply:
I took the name in the documentation https://pm2.keymetrics.io/docs/usage/application-declaration/